tests/sentry/api/endpoints/test_sudo.py

```diff
@@ -29,9 +29,9 @@ def test_sudo_required_del_org(self):
         with self.settings(MIDDLEWARE_CLASSES=tuple(middleware)):
             response = self.client.delete(url, is_sudo=False)
             assert response.status_code == 401
-            assert response.data['detail']['code'] == 'sudo-required'
-            assert response.data['detail']['message'] == 'Account verification required.'
-            assert response.data['detail']['extra']['username'] == 'foo@example.com'
+            assert response.data['code'] == 'sudo-required'
+            assert response.data['message'] == 'Account verification required.'
+            assert response.data['extra']['username'] == 'foo@example.com'
```

Comment:
Does the frontend need to be updated for these changes?

Reply:
Damn, yeah. I'll see if I can figure out why this change is actually happening. For most api responses the `detail` level is maintained, but not for this middleware.

Reply:
@markstory Resolved this, good catch. My changes should keep our responses the same as they were before.